test/processors/image_processor_test.rb

```diff
@@ -22,9 +30,16 @@ class ImageProcessorTest < ActiveSupport::TestCase
       }.with_indifferent_access
     }
 
-    it "should resize a local file" do
+    it 'should resize a local file' do
+      if travis?
+        audio_monster.expect(:create_temp_file, Tempfile.new('test'), [String])
+        audio_monster.expect(:create_temp_file, Tempfile.new('test'), [String])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.